### ISSUE TITLE
High training accuracy but low validation accuracy?

### ISSUE THREAD

**Issue by EasonD3** [dmlc/mxnet#2147]
I'm fine-tuning a model (based on Inception-BN) by using a image dataset with 10 classes (my case is a multi-class problem). I randomly split the whole dataset into training and test subsets. During the training phase, I observe that the training accuracy is very high (i.e., > 90%), while the validation accuracy is rather low (i.e., ~20%).

I also tried to use the model to predict tens of test images. It seems the prediction scores of the 10 classes are roughly the same for each test image, i.e., all the test images were predicted to be of the same class.

I've double checked the `.rec` files I generated. They are good. And also, for fine-tuning, I simply copied the Inception-BN symbol file into a new one and modified the last fully connected layer to have its `num_hidden=10` and its name renamed. Things look pretty good to me.. (what I did is very similar to fine-tuning in `Caffe`, i.e., only need to rename the fully connected layer and reset its `num_output` in `train_val.prototxt`).

Did anyone encounter the same problem before? Thanks.

**Comment by EasonD3** [comment]
The stated issue is due to the pre-trained Inception-BN model. Please see #2155 for the solution.
